values.yaml

```diff
@@ -53,6 +53,13 @@ kpi:
     minAvailable: 1
   env:
     normal:
+      # By default, the following are set but can be overridden
+      DEFAULT_FROM_EMAIL: "support@kobotoolbox.org"
```

Comment:
Let's omit this for a few reasons:
* to help keep down the load of emails that our support team has to handle;
* to avoid random instances unintentionally sending emails about account activation, password reset, REST Services, etc. using our official email address.